tests/test_comparison.py

```diff
@@ -6,12 +6,12 @@
 evap = read_csv("tests/data/evap.csv", index_col=0, parse_dates=True).squeeze("columns")
 obs = read_csv("tests/data/obs.csv", index_col=0, parse_dates=True).squeeze("columns")
 
-ml1 = ps.Model(obs, name="Test_Model")
+ml1 = ps.Model(obs.dropna(), name="Test_Model")
```

Comment:
remove dropnas?

Reply:
I will keep it like this, as the purpose of the test is not to test if Model works for oseries with nan-values.